Applications/shapeworks/MeshCommands.cpp

```diff
@@ -261,6 +261,35 @@ bool Decimate::execute(const optparse::Values &options, SharedCommandData &share
   return sharedData.validMesh();
 }
 
+///////////////////////////////////////////////////////////////////////////////
+// CVDDecimate
+///////////////////////////////////////////////////////////////////////////////
+void CVDDecimate::buildParser()
+{
+  const std::string prog = "cvd-decimate";
+  const std::string desc = "applies cvd decimation filter";
```

Comment:
add "(centroidal voronoi diagram)" as with the other help and docs

Reply:
Done